src/polyfills/fetch.js

```diff
@@ -68,9 +68,9 @@ export function fetchPolyfill(input, init = {}) {
     }
 
     if (init.headers) {
-      Object.keys(init.headers).forEach(function (header) {
+      for (const header of Object.keys(init.headers)) {
```

Comment:
Fetch polyfill is explicitly for IE11, so let's undo this.

Reply:
Reverted this file.